Modules/ZDC/src/ZDCRawDataTask.cxx

```diff
@@ -453,6 +459,29 @@ void ZDCRawDataTask::initHisto()
   } else
     setBinHisto1D(26, -0.5, 25.5);
   addNewHisto("SUMMARYRATE", "hrateSummary", "Rate Summary (KHz)", "NONE", "LBC");
+
+  if (auto param = mCustomParameters.find("SUMMARY_ALIGN"); param != mCustomParameters.end()) {
+    ILOG(Debug, Devel) << "Custom parameter - SUMMARY_ALIGN: " << param->second << ENDM;
+    tokenString = tokenLine(param->second, ";");
+    setBinHisto2D(atoi(tokenString.at(0).c_str()), atof(tokenString.at(1).c_str()), atof(tokenString.at(2).c_str()), atoi(tokenString.at(3).c_str()), atof(tokenString.at(4).c_str()), atof(tokenString.at(5).c_str()));
+  } else
```

Comment:
we strongly recommend to use curvy braces even for single line instructions.

Reply:
ok I change it